src/browser_action.js

```diff
@@ -450,6 +450,15 @@ browseraction.createEventDiv_ = function(event) {
 
   }
 
+  if (event.attachments) {
```

Comment:
Your PR comment is worth adding here inline as well!
“If there are multiple attachments, do nothing. This ideally would have a nice UI, but I think it's OK because multiple attachments are the exception.”

Reply:
Thanks for the review! Done.